src/Tizen.NET.MaterialComponents/Components/MSheets.cs

```diff
@@ -0,0 +1,130 @@
+using ElmSharp;
+using System.Collections.Generic;
+
+namespace Tizen.NET.MaterialComponents
+{
+    public class MSheets : MNavigationDrawer
+    {
+        double _ratio;
+        double _sideSheetsDefaultRatio = 0.8444;
+        double _bottomSheetsDefaultRatio = 0.5;
+        double _bottomSheetsMinRatio = 0.084;
+        double _bottomSheetsMaxRatio = 0.5;
+
+        Panel _panel;
+        EvasObject _backSheets;
+        MNavigationView _nv;
+        EvasObject _frontSheets;
+        List<MItem> _frontList;
+
+        public MSheets(EvasObject parent, SheetsDirection direction) : base(parent)
+        {
+            _panel = base._drawer;
+            _nv = new MNavigationView(parent);
+            base.NavigationView = _nv;
+            UpdateDirection(direction);
+        }
+
+        public EvasObject BackSheets
```

Comment:
why not public `EvasObject BackSheets => Main = _backSheets;`

Reply:
ok